spec/ruby/core/queue/initialize_spec.rb

```diff
@@ -0,0 +1,50 @@
+require_relative '../../spec_helper'
+
+describe "Queue#initialize" do
+  it "can be passed no arguments for an empty Queue" do
+    q = Queue.new
+    q.size.should == 0
+    q.empty?.should be_true
```

Comment:
```suggestion
    q.should.empty?
```

for better failure message, and also other `be_true`/`be_false` calls.